openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -276,23 +277,40 @@ This profile mandates the support for X.509 certificate-based key resolution to
 
 * If the credential has cryptographic holder binding, a KB-JWT, as defined in [@!I-D.ietf-oauth-sd-jwt-vc], MUST always be present when presenting an SD-JWT VC.
 
-# Crypto Suites
+# Crypto Suites {#crypto-suites}
 
 Cryptography is required by the following operations:
 
-- to sign and validate the signature on the Wallet Attestation and its proof of possession
+- to sign and validate the signature on the Wallet or Key Attestation, and their proof of possession (PoP)
+- to sign and validate the signature of the status information of the Verifiable Credential or Wallet Attestation
 - to sign and validate the Issuer's signature on the Verifiable Credential
 - to sign and validate the Holder's signature on the Verifiable Presentation
 - to sign and validate the Verifier's signature on the Presentation Request
```

Comment:
Perhaps we can just remove this, since we are repeating this below.
```suggestion
```

Reply:
agreed.